palette/src/rgb/rgb.rs

```diff
@@ -2,6 +2,7 @@ use core::any::TypeId;
 use core::fmt;
 use core::marker::PhantomData;
 use core::ops::{Add, AddAssign, Div, DivAssign, Mul, MulAssign, Sub, SubAssign};
+use std::str::FromStr;
```

Comment:
This need to be changed to get it from `core` instead, for the `#[no_std]` support to work.

```suggestion
use core::str::FromStr;
```